### PR TITLE
refactor(rust): remove default value from help text

### LINKED ISSUE #3414: Remove generated name in `ockam space create` help
# Currently

Note the generated name next to the NAME argument 
Screen Shot 2022-09-03 at 3 07 56 AM

# Desired

We don't want to display the generated value (ex: e8218333) in help.

The code of `ockam space create` command is here
https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/space/create.rs

### PR DESCRIPTION
**Pull request by RajivTS**
`ockam space create` currently outputs helptext that includes the default value
for the space name which is a randomly generated hex. This PR removes that default
value from help text.

closes #3414

### CODE REVIEW

**Review by mrinalwadhwa — APPROVED** [review]
That looks great. Thank you!

**Comment by mrinalwadhwa** [comment]
@RajivTS thank you for an awesome first contribution 🥳 
This looks ready to merge after we land the [CLA PR](https://github.com/build-trust/ockam-contributors/pull/103)

**Comment by RajivTS** [comment]
Closing this PR since it contains unsigned commits. Created another in its place #3425.

**Pull request closed without merging**
